2020Labs/CustomSecurityContent/ansible/configs/rhel-custom-security-content/labs_setup.yml

```diff
@@ -3,7 +3,7 @@
 # Run me as a verification-only playbook:
 # ansible-playbook ... -e VERIFICATION_RUN=true --check --diff
 
-- hosts: all
```

Comment:
What is the reason for this limitation? I think that this limits the sets of hosts as provided by the inventory.
In other words, you won't be able to apply this playbook to a (local) VM without an inventory file listing the VM's hostname as part of the `nodes` group.
See also https://stackoverflow.com/questions/33222641/override-hosts-variable-of-ansible-playbook-from-the-command-line
https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#id19